utils/filesystem/files.go

```diff
@@ -1129,6 +1128,60 @@ func (fs *VFS) LsWithExclusionPatterns(dir string, exclusionPatterns ...string)
 	return
 }
 
+func (fs *VFS) LsRecursive(ctx context.Context, dir string) (files []string, err error) {
+	limits := NoLimits()
+	return fs.LsRecursiveWithExtensionPatternsAndLimits(ctx, dir, limits)
```

Comment:
just do this inline
```suggestion
	return fs.LsRecursiveWithExtensionPatternsAndLimits(ctx, dir, NoLimits())
```

Reply:
I think it only makes sense to instantiate a lone variable if it is being used in multiple places